wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Create/ConversationCreationController.swift

```diff
@@ -130,10 +130,6 @@ final class ConversationCreationController: UIViewController {
 
     // MARK: - Life cycle
 
-    convenience init() {
-        self.init(preSelectedParticipants: nil, selfUser: ZMUser.selfUser())
-    }
```

Comment:
🤘

Reply:
Did we use that init anywhere?

Reply:
Yes, I had to change one line in `StartUIViewController+SearchResults.swift` in the method `func openCreateGroupController()`.